{"name":"call_user_func","purpose":"Call a user function given by the first parameter","manualid":"function.call-user-func","version":"PHP 4, PHP 5","params":[{"name":"function","type":"callback","optional":"false"},{"name":"parameter","type":"mixed","optional":"true"},{"name":"...","type":"mixed","optional":"true"}],"return":{"type":"mixed","description":"Returns the function result, or FALSE on error."},"errors":null,"notes":[{"type":"note","description":"Callbacks registered\nwith functions such as call_user_func and call_user_func_array will not be\ncalled if there is an uncaught exception thrown in a previous callback."}],"changelog":[],"seealso":[{"type":"function","name":"call_user_func_array"},{"type":"function","name":"is_callable"},{"type":false,"name":false}]}